### ISSUE TITLE
Ticket creation fails when field 'components' cannot be set

### ISSUE THREAD

**Issue by bornmanm** [free/jiralert#7]
Hi,

creating a ticket fails with the error message

`Field 'components' cannot be set. It is not on the appropriate screen, or unknown.`

Due to Jira permissions I do not have the permission to edit the corresponding screen to allow the field 'components' to be set. Uncommenting the lines 

```
issue.Fields.Components = make([]*jira.Component, 0, len(r.conf.Components))
for _, component := range r.conf.Components {
    issue.Fields.Components = append(issue.Fields.Components, &jira.Component{Name: component})
}
```
from notify.go solves the problem but it should probably be configurable whether jiralert attempts to set 'components'.

**Comment by free** [comment]
I've wrapped the offending code in an `if len(r.conf.Components) > 0` block, so it will now no longer set an empty list of components. I'll try to make a new release soon, but I'm trying to get the other 2 issues solved as well.

**Comment by free** [comment]
Closing as fixed, let me know if you still have problems. Fix is part of [release 0.6](https://github.com/free/jiralert/releases/tag/0.6).

**Comment by bornmanm** [comment]
Everything is working fine now. Thanks a lot!
